Src/BootCamp.Chapter/Program.cs

```diff
@@ -2,11 +2,99 @@
 
 namespace BootCamp.Chapter
 {
+    class Person
+    {
+        private string _name;
+        private string _age;
+        private string _weight;
+        private string _height;
+
+
+        public string Name {
+            get
+            {
+                return _name;
+            }
+            set
+            {
+                _name = value;
+            } 
+        
+        }
+
+        public string Age
+        {
+            get
+            {
+                return _age;
+            }
+            set
+            {
+                _age = value;
+            }
+
+        }
+
+        public string Weight {
+            get
+            {
+                return _weight;
+            }
+            set
+            {
+                _weight = value;
+            }
+        }
+
+        public string Height
+        {
+            get
+            {
+                return _height;
+            }
+            set
+            {
+                _height = value;
+            }
+
+        }
```

Comment:
This can be auto properties.
Instead of:
```cs
        public string Name {
            get
            {
                return _name;
            }
            set
            {
                _name = value;
            } 

        }
```
you can:
```cs
        public string Name{get;set;}
```